nvflare/app_common/executors/script_executor.py

```diff
@@ -70,6 +70,7 @@ def __init__(
             params_transfer_type=params_transfer_type,
```

Comment:
This script_executor still has the import from nvflare.app_opt. And it has conditional import from "pt" or "tf". We should break them up into separate files.

Reply:
If we break up ScriptExecutor into separate PT and TF versions, then `PTScriptExecutor` will be exactly the same as `PTInProcessClientAPIExecutor`, should we just use that instead then?

Reply:
I don't think issues about ScriptExecutor belong to this PR.
Please open a different PR to address the issues, if you think there are issues.
To me, the only issue is that this module should be in app_opt. But let's address this in a different PR.